doc/examples/scraper/README.md

```diff
@@ -1,6 +1,6 @@
 # Quick Start Guide: Web Scraper
 In this guide you're going to create a Pachyderm pipeline to scrape web pages. 
-We'll use a standard unix tool, `wget`, to do our scraping. 
+We'll use a standard unix tool, `wget`, to do our scraping.
```

Comment:
This is a nitpick, but does affect readability of our PRs.

It looks like your editor removed trailing whitespace from a bunch of lines. That makes this diff quite a bit more noisy. Can you re-submit without this change? 

Maybe its time for us as a team to have some conventions around a few editor settings (wrt whitespace / tabs v spaces).